frontend/src/components/task/task-item.js

```diff
@@ -26,7 +26,7 @@ class TaskItem extends Component {
   componentDidMount () { }
 
   handleClickListItem (id) {
-    this.props.history.push('/task/' + id)
+    this.props.history.push('/task/' + id + '/gitpay')
```

Comment:
What is this? Why are you hardcoding `/gitpay` here?

Reply:
Sorry. Wasn't need to be there.